<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>881</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Данные</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>227</x>
        <y>39</y>
        <width>161</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>39</y>
        <width>181</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>9</y>
        <width>151</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Поиск по фамилии</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>130</y>
        <width>161</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортиовать по дате</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>80</y>
        <width>641</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>180</y>
        <width>161</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addAtPositionButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>230</y>
        <width>161</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить на позицию</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>370</y>
        <width>161</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>280</y>
        <width>161</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>330</y>
        <width>161</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить всё</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>работа с файлом</string>
     </attribute>
     <widget class="QPushButton" name="loadButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>741</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Загрузить файл</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="longestWordLineEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>421</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="countLineEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>421</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>211</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Вывод самог длинного слова</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>281</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>сколько раз это слово встретилось в файле</string>
      </property>
     </widget>
     <widget class="QPushButton" name="analyzeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>741</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Анализ</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
